### DISCUSSION TITLE
Confusion over 'release regex'

### DISCUSSION THREAD

**Discussion by jimmcslim** [community/community#12873] [Other Feature Feedback, Questions, & Ideas] [answered]
[This answer](https://github.com/github/feedback/discussions/5975#discussioncomment-1799743) attempts to provide a solution to some auto-generated release notes picking up the incorrect base tag.

> We've just rolled out the new release regex to all repositories. Please take a look and see if it fixes your issues.

But it's not clear to me where this 'release regex' is?

Also see my comment on that issue https://github.com/github/feedback/discussions/5975#discussioncomment-2352770

**Comment by MylesBorins** [comment, marked as answer]
The regex is an internal implementation detail, not something exposed via config. Apologies for the confusion

**Reply by panjacek** [reply]
it would be nice if "last" release to compare could be simply dropdown just in case auto regex fails to detect it.
